client/src/main/logic/savingTimer.ts

```diff
@@ -17,6 +17,7 @@ const timeToWait = 10 * 1000;
 /**
  * The store for the callbacks
  */
+// eslint-disable-next-line
```

Comment:
Why did this need to have eslint disabled?

Reply:
It was complaining about the parameter being a Function. I put that in during the merge to make sure it wasnt a problem and forgot about it. It compiles fine so ill get rid of it.

"Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape.eslint@typescript-eslint/ban-types"

Reply:
Also, thet checks that run on committing dont pass without that. I bypassed the check just to get it up.
![Capture --no-verify](https://user-images.githubusercontent.com/46167982/86712619-08ce5a00-bfd2-11ea-9f76-e46e80788664.PNG)

Reply:
It compiles and runs OK. The TravisCI checks all pass... I don't get it.

Reply:
Oh. Strange that it was running locally on this end without throwing anything on that one. Not sure what happened there.